ui/components/component-library/banner-base/banner-base.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { ButtonIcon, ICON_NAMES, Text } from '..';
+import Box from '../../ui/box';
+
+import {
+  COLORS,
+  DISPLAY,
+  SIZES,
+  TEXT,
+} from '../../../helpers/constants/design-system';
+
+export const BannerBase = ({
+  className,
+  title,
+  description,
+  action,
+  leftAccessory,
+  onClose,
+  closeButtonProps,
+  ...props
+}) => {
+  return (
+    <Box
+      className={classnames('mm-banner-base', className)}
+      display={DISPLAY.INLINE_FLEX}
+      gap={2}
+      backgroundColor={COLORS.BACKGROUND_DEFAULT}
+      borderRadius={SIZES.SM}
+      padding={3}
+      {...props}
+    >
+      {leftAccessory && (
+        <div className="mm-banner-base__left-accessory">{leftAccessory}</div>
+      )}
+
+      <div className="mm-banner-base__">
+        {title && (
+          <Text className="mm-banner-base__title" variant={TEXT.BODY_LG_MEDIUM}>
+            {title}
+          </Text>
+        )}
+        {description && (
+          <Text className="mm-banner-base__description">{description}</Text>
+        )}
+        {action && <Text className="mm-banner-base__action">{action}</Text>}
+      </div>
```

Comment:
Can we use fragment here?
```suggestion

      <>
        {title && (
          <Text className="mm-banner-base__title" variant={TEXT.BODY_LG_MEDIUM}>
            {title}
          </Text>
        )}
        {description && (
          <Text className="mm-banner-base__description">{description}</Text>
        )}
        {action && <Text className="mm-banner-base__action">{action}</Text>}
      </>
```